behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/tilrettelegging/OppholdÅrsak.java

```diff
@@ -0,0 +1,6 @@
+package no.nav.foreldrepenger.behandlingslager.behandling.tilrettelegging;
+
+public enum OppholdÅrsak {
```

Comment:
Gjerne SvpOppholdÅrsak så slipper man kampen med OppholdÅrsak for FP i søknad, fordeling og uttak :-)